radis/lbl/loader.py

```diff
@@ -1392,23 +1434,39 @@ def fetch_databank(
             )
 
         # Always sort line database by wavenumber (required to SPARSE_WAVERANGE mode)
-        df.sort_values("wav", ignore_index=True, inplace=True)
+        if output == "pandas":
+            df.sort_values("wav", ignore_index=True, inplace=True)
+        elif output == "vaex":
+            try:
```

Comment:
why is rewriting the attrs needed ? add comment

Reply:
Because dataframe returned by sort_values doesn't have attrs, so I have to add it again.